README.rst

```diff
@@ -132,15 +132,15 @@ project that you might want to use:
 - `ptpython and ptipython <https://pypi.org/project/ptpython/>`
 - `xonsh <https://xon.sh/>`
 
-Ignoring commits with git blame.ignoreRevsFile
+Ignoring commits with the git blame.ignoreRevsFile
 ==============================================
 
 As of git 2.23, it is possible to make formatting changes without breaking
 ``git blame``. See the `git documentation
 <https://git-scm.com/docs/git-config#Documentation/git-config.txt-blameignoreRevsFile>`_
 for more details.
 
-To use this feature you must:
```

Comment:
That is incorrect english, you "must install" and "must configure", not "must do install" and "must do configure".